### DISCUSSION TITLE
Why is the API document removed from the official website page

### DISCUSSION THREAD

**Discussion by bobo0810** [hpcaitech/ColossalAI#2937] [Community | Q&A]
Why is the API document removed from the official website page？

**Comment by FrankLeeeee** [comment]
Hi, we are integrating the API documentation into the our website without directing to Read the Docs.

**Reply by bobo0810** [reply]
The api document has not been updated for nearly two months, so I would like to confirm whether the api document will not be provided in the future.

**Reply by binmakeswell** [reply]
> The api document has not been updated for nearly two months, so I would like to confirm whether the api document will not be provided in the future.

This has always been on our TODO list, but with limited manpower at the moment, we have to deal with other high priority items first. Sorry about it.
